dmoj/cli.py

```diff
@@ -316,9 +316,10 @@ def execute(self, line):
             print ansi_style('#ansi[%s](yellow)/#ansi[%s](green) in %s' % (problem, i + 1, lang))
         print
         
+
 class DifferenceCommand(Command):
-    name: 'diff'
-    help: 'Shows difference between two files.'
+    name = 'diff'
+    help = 'Shows difference between two files.'
```

Comment:
Should also allow showing difference between two submissions, or a mix of submissions and files (if argument is an integer, it can be assumed to be a submission).

If `git` exists in `PATH`, selected submissions would be written to a temp file [`tempfile`](https://docs.python.org/2/library/tempfile.html) module before being passed to `git diff`.